crates/hc_deepkey_types/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "hc_deepkey_types"
+version = "0.8.0-dev.2"
+authors = ["Matthew Brisebois <matthew.brisebois@holo.host>"]
+edition = "2021"
+license = "CAL-1.0"
+repository = "https://github.com/holochain/holochain"
+documentation = "https://docs.rs/hc_deepkey_types"
+description = "Definitions used by the Deepkey DNA Zomes"
+readme = "README.md"
+
+[lib]
+name = "hc_deepkey_types"
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+# TODO: remove path designation to allow this crate to trail behind the current monorepo version
+hdi = { version = "=0.5.0-dev.11", path = "../hdi" }
```

Comment:
Does this need to be pinned? We generally let the release automation bump dependencies and just use the latest version of everything.